opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -775,17 +775,17 @@ namespace detail {
         asImpl().assembleMassBalanceEq(state);
 
         // -------- Well equations ----------
-
+        IterationReport iter_report;
```

Comment:
This must be initialized, especially the well_iterations member must be set to zero. Otherwise the return below will give garbage.